### ISSUE TITLE
Accessibility: Default cleanwhite theme contains a lot of color contrast violations

### ISSUE THREAD

**Issue by sandbergja** [subjectsplus/SubjectsPlus#1370]
WebAIM has a helpful color contrast checker: https://webaim.org/resources/contrastchecker/

See also their article on the topic: https://webaim.org/articles/contrast/
